src/utils/formatNumbers.ts

```diff
@@ -436,6 +436,30 @@ function useFormatterLocales(): {
   }
 }
 
+interface FormatCurrencyAmountOptions {
+  amount: Nullish<CurrencyAmount<Currency>>
+  type?: NumberType
+  placeholder?: string
+  locale?: SupportedLocale
+  localCurrency?: SupportedLocalCurrency
+}
+
+export function formatCurrencyAmount({
```

Comment:
nit: could you add a description to this function and to `useFormatter` detailing what it does and how to use it?

Reply:
will do that in the base pr